src/components/Link/Link.scss

```diff
@@ -30,6 +30,12 @@
   overflow: inherit;
   text-overflow: inherit;
 
+  &.is-disabled {
+    color: $ms-color-neutralTertiary;
+    pointer-events: none;
+    cursor: pointer;
```

Comment:
cursor: pointer is probably not wanted here. That makes disabled links look clickable...